HISTORY.md

```diff
@@ -8,6 +8,7 @@
 * Subcompactions are now disabled when user-defined timestamps are used, since the subcompaction boundary picking logic is currently not timestamp-aware, which could lead to incorrect results when different subcompactions process keys that only differ by timestamp.
 
 ### New Features
+* ldb now has a new feature, `list_live_files_metadata`, which lists all the live SST files, their column families and the LSM level they are located at.
```

Comment:
Branch is outdated because 6.22 branch has been cut. You can add unreleased and New feature  section and update your change there.

Reply:
Oh I see this is subtle, thanks for the catch. There was already an "Unreleased" section I didn't realize that there was no 6.22 yet. Adding a new unreleased in the next push.